src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/ConditionPatternParser.cs

```diff
@@ -217,7 +216,7 @@ public static ParsedModRewriteInput ParseProperty(ParserContext context, bool in
             }
         }
 
-        private static bool IsValidActionCondition(ParsedModRewriteInput results)
+        private bool IsValidActionCondition(ParsedModRewriteInput results)
```

Comment:
Same as above. If the function behave the same with and without `static` modifier, leave it on.